src/datreant/tests/test_treants.py

```diff
@@ -367,14 +367,26 @@ def test_add_categories(self, treant):
             treant.categories.add({'bark': 'snark'}, lark=27)
             assert 'bark' in treant.categories
             assert 'snark' not in treant.categories
-            assert 'bark' in treant.categories
+            assert 'lark' in treant.categories
 
             assert treant.categories['bark'] == 'snark'
             assert treant.categories['lark'] == 27
 
             treant.categories['lark'] = 42
             assert treant.categories['lark'] == 42
 
+            treant_tuple = ('ark', 'clark', 1)
+            treant.categories['hark'] = treant_tuple
+            assert treant.categories['hark'] == list(treant_tuple)
```

Comment:
Looks like this is necessary, because the value would always be returned as a `list` and never as a `tuple`.

Reply:
This is probably because json doesn't distinguish between list and tuple, so it gets squashed when you bounce it off the json file